eslint.config.mjs

```diff
@@ -0,0 +1,36 @@
+import hmppsConfig from '@ministryofjustice/eslint-config-hmpps'
+
+export default [
+  // shared HMPPS standards
+  ...hmppsConfig,
+  
+  // app-specific overrides
+  {
+    ignores: ['**/node_modules', 'dist/', 'test_results'],
+  },
+  {
+    languageOptions: {
+      globals: {
+        ...hmppsConfig.find(obj => Object.keys(obj).includes('languageOptions')).languageOptions.globals,
+        $: false,
+      },
+    },
+  },
+  {
+    rules: {
+      'import/no-extraneous-dependencies': [
+        'error',
+        {
+          devDependencies: [
+            '**/*.test.ts',
+            'integration_tests/**',
```

Comment:
needed to add `integration_tests` here otherwise it would moan about files inside (including `*.spec.ts` files) referencing test dependencies